app/components/point.jsx

```diff
@@ -20,6 +23,10 @@ const Point = forwardRef((props, ref) => {
     setIsHovered(false)
   }
 
+  const handleLoading = () => {
```

Comment:
I don't know when handleLoading would get called to reset the isLoading.  If you have an idea lets discuss.

I was thinking there would be a vState for 'loading'. that the parent would change, but that's not very clean.

The use case is that first the page loads, but there is no data.  Later on the data fetch completes and the data is there.

Reply:
I'm not sure yet either. This is the main issue for me